packages/cli/src/actions/init.ts

```diff
@@ -125,10 +131,15 @@ const init: InitCommandHandler = async ({ client, beta, tuna }) => {
         spaces: 2
       });
     }
-
+    let daemon = daemonPackage;
+    let board = boardPackage;
+    if (version) {
+      daemon += `@${version}`;
+      board += `@${version}`;
```

Comment:
As @liangzr mentioned at #446, using the same version for daemon and board would cause that we need to bind the board & daemon release. I have no idea about this until we publish the board online.

Reply:
How about split the board version from this PR?

Reply:
Do you mean the init action don't install the pipboard?

Reply:
It depends on the rule of dependence between their versions, maybe we should add a `peerDependencies` in `daemon/package.json`, and check it in CI (as a precondition for Docker building)